### DISCUSSION TITLE
Is there any way to integrate the superset dashboard into the Angular 12 application? If yes can I have some reference?

### DISCUSSION THREAD

**Discussion by alitariqse** [apache/superset#19649] [Q&A / Help]
Hi, 
Hope you all are doing great. 
I have an Application in Angular 12. I am doing R&D for integrating superset dashboard into my application but not getting any help and support regarding this. I've been trying for many days.
Please let me know if there is any way to integrate Dashboard into my Angular application. So I can Save my time. 

Thanks & Best Regards
Ali Tariq Mirza

**Comment by srinify** [comment]
Hey @alitariqse moving to Discussions as Issues is focused on bugs and SIPs! You're also welcome to ask in the Superset Slack community!

**Comment by kpolyd** [comment]
Hello, 
I'm also looking to integrate Superset dashboards and charts in Angular. 
@alitariqse Do you have an update?

**Reply by zs-err** [reply]
Di you have any updates on this?
